font/src/ft/mod.rs

```diff
@@ -91,7 +94,9 @@ pub struct FreeTypeRasterizer {
     faces: HashMap<FontKey, FaceLoadingProperties>,
     ft_faces: HashMap<FTFaceLocation, Rc<FTFace>>,
     fallback_lists: HashMap<FontKey, FallbackList>,
+    keys: HashMap<PathBuf, FontKey>,
```

Comment:
So this one has undergone multiple changes... and is not in master anymore (https://github.com/alacritty/alacritty/commit/15cc07c069b09f109ed18fb94e02e9650be7fa33 -> https://github.com/alacritty/alacritty/commit/2672770683244bdfb55273a11328b874e2d2af04 -> https://github.com/alacritty/alacritty/commit/84f57ac8367e3e24e0347515e6681c0478fc3d29)

```suggestion
```